setup.py

```diff
@@ -7,22 +7,23 @@
     version=__version__,
     description="Mapping GDC's schema to Ellrot's lab FHIR schema.",
     long_description=open('README.md').read(),
-    # url='',
-    author='https://ellrottlab.org/',  
+    url='https://github.com/bmeg/gdc2fhir',
+    author='https://ellrottlab.org/',
     packages=find_packages(),
-    """
     entry_points={
-        'console_scripts': []
+        'console_scripts': ['gdc2fhir = gdc2fhir.cli:cli']
     },
```

Comment:
@bwalsh ➜ /workspaces/gdc2fhir (mappings) $ pip freeze | grep gdc
gdc2fhir==0.0.0
@bwalsh ➜ /workspaces/gdc2fhir (mappings) $ gdc2fhir
Traceback (most recent call last):
  File "/home/codespace/.local/bin/gdc2fhir", line 33, in <module>
    sys.exit(load_entry_point('gdc2fhir==0.0.0', 'console_scripts', 'gdc2fhir')())
  File "/home/codespace/.local/bin/gdc2fhir", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "/opt/python/3.10.13/lib/python3.10/importlib/metadata/__init__.py", line 171, in load
    module = import_module(match.group('module'))
  File "/opt/python/3.10.13/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'gdc2fhir.__main__'
@bwalsh ➜ /workspaces/gdc2fhir (mappings) $